### ISSUE TITLE
Mods not loading

### ISSUE THREAD

**Issue by Kitsudragon** [TanninOne/modorganizer#774]
First, will this mod manager support a pre-1.5 game? That game doesn't recognize any mods, but I don't know if that's expected at this point, or not.

The post 1.5 game loads... but I have a radio mod loaded and it isn't visible in the Pip boy. So it might not be loading those either. Any suggestions?

**Comment by rdunlap** [comment]
Patch Level 1.5 changed some basic stuff and now MO2 has had to change to the new format. So, no pre 1.5 patch levels will not work with it.

Robert

**Comment by Kitsudragon** [comment]
Do I have to register with Bethesda through the mod add-in option to use mods at all now? I've gotten the MO program to start with F4SE just fine, but none of the mods seem to be working. I haven't used any of the mod add-in functions in the main menu largely because I don't know if they'll conflict with the rest of this.

**Comment by TanninOne** [comment]
I won't add pre-1.5 support.

When you have trouble with a specific mod, please always post a link to it!
And: please create one report per issue

**Comment by Kitsudragon** [comment]
Okay, I started clean with all the mods, d/loaded all the new ones where applicable, and started the program.

The first time I tried, it seemed to be working. I got a brief message about "building vfs" and then the game started. Loading a new game was where it stopped. The game loading screen started to chunk down, updating a single frame every thirty to forty seconds. Finally, it just became a gray screen with "Fallout 4 has stopped working." Every time since then I've tried to start the game, either through the program or the shortcut, I've gotten the same gray screen message.
